reactor-test/src/main/java/reactor/test/StepVerifier.java

```diff
@@ -1021,28 +1021,6 @@ default Duration verifyTimeout(Duration duration) {
 		 */
 		Step<T> expectNoFusionSupport();
 
-		/**
```

Comment:
there was no "to be removed" even though this one has been deprecated since 3.3.1... I wonder if there could be anybody out there testing that no `onSubscribe` happens for a given duration (which would be a legitimate-ish verification). Can restore it for now and open an issue for that specific case to remove the method and add a suitable alternative in a follow-up PR ?

Reply:
But no `onSubscribe` after `subscribe()` would be a spec violation, isn't it?

Reply:
yes, but a _delayed_ onSubscribe wouldn't.

Reply:
Is it so?
https://github.com/reactive-streams/reactive-streams-jvm/issues/486

Reply:
right... used to be a gray area. let's remove it then, even though the `@Deprecated` was intended as a deterrent to using that method without at least thinking about calling `expectSubscription()`.

Reply:
Ok, will re-remove it :D 👍